<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerClass</class>
 <widget class="QMainWindow" name="ViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>体数据</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="volumeAdd">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="volumeList"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>二维图像</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="imageAdd">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="imageList"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="imageLayout"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="image_origin_mode">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>原始</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="image_bin_mode">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>二值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="image_fuse_mode">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>融合</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="vtkLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRegister"/>
   <addaction name="actionFuse"/>
  </widget>
  <action name="actionRegister">
   <property name="icon">
    <iconset>
     <normaloff>Resources/register.png</normaloff>Resources/register.png</iconset>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
   <property name="toolTip">
    <string>配准</string>
   </property>
  </action>
  <action name="actionFuse">
   <property name="icon">
    <iconset>
     <normaloff>Resources/fuse.png</normaloff>Resources/fuse.png</iconset>
   </property>
   <property name="text">
    <string>Fuse</string>
   </property>
   <property name="toolTip">
    <string>融合</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Viewer.qrc"/>
 </resources>
 <connections/>
</ui>
